cmd/operator-sdk/add/controller.go

```diff
@@ -48,19 +54,36 @@ Example:
 	│   └── appservice_controller.go
 	└── controller.go
 
+The following example will create a controller to manage, watch and reconcile as a primary resource a resource which is 
+not defined in the project (external). Note that, it can be used to create controllers for any External API. In this 
+case, it will be done for the resource "v1.Deployments" from the Kubernetes API. 	
+
+Example:
+
+	$ operator-sdk add controller  --api-version=k8s.io.api/v1 --kind=Deployment  --custom-api-import=k8s.io/api/apps
+	$ tree pkg/controller
+	pkg/controller/
+	├── add_deployment.go
+	├── deployment
+	│   └── deployment_controller.go 
+	└── controller.go
 `,
 		RunE: controllerRun,
 	}
 
-	controllerCmd.Flags().StringVar(&apiVersion, "api-version", "", "Kubernetes APIVersion that has a format of $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1)")
+	controllerCmd.Flags().StringVar(&apiVersion, "api-version", "",
+		"Kubernetes APIVersion that has a format of $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1)")
 	if err := controllerCmd.MarkFlagRequired("api-version"); err != nil {
 		log.Fatalf("Failed to mark `api-version` flag for `add controller` subcommand as required")
 	}
-	controllerCmd.Flags().StringVar(&kind, "kind", "", "Kubernetes resource Kind name. (e.g AppService)")
+	controllerCmd.Flags().StringVar(&kind, "kind", "",
+		"Kubernetes resource Kind name. (e.g AppService)")
 	if err := controllerCmd.MarkFlagRequired("kind"); err != nil {
 		log.Fatalf("Failed to mark `kind` flag for `add controller` subcommand as required")
 	}
-	controllerCmd.Flags().StringVar(&customAPIImport, "custom-api-import", "", `External Kubernetes resource import path of the form "host.com/repo/path[=import_identifier]". import_identifier is optional`)
+	controllerCmd.Flags().StringVar(&customAPIImport, "custom-api-import", "",
+		`The External API import path of the form "host.com/repo/path[=import_identifier]" which will be managed 
+	by the controller. Note that import_identifier is optional. ( E.g. --custom-api-import=k8s.io/api/apps )`)
```

Comment:
This multi-line help message renders a little weird for me. I think I'd suggest keeping it all on one line and letting things just wrap, but open to other suggestions. Another option would be to stick to 80 characters and use spaces instead of a tab at the beginning of the line to ensure things align the same on all terminals.